data/transition-sites/directgov_los.yml

```
@@ -0,0 +1,6 @@
+---
+site: directgov_los
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
```

The README says this field should be:

> URL for new site, used to redirect '/'. Must include the 'http' or 'https'.

So shouldn't this be as follows:

```suggestion
homepage: https://find-your-nearest-jobcentre.dwp.gov.uk
```

---

Fair enough - this is auto-populated in a rather unhelpful way by the rake task :-|.